src/Resources/ResourceManager/Implementation/Policy/PolicyHelpStrings.cs

```diff
@@ -35,6 +35,7 @@ public static class PolicyHelpStrings
         public const string NewPolicyAssignmentPolicyParameterObjectHelp = "The policy parameters object for the new policy assignment.";
         public const string NewPolicyParameterHelp = "The policy parameters file path or string for the new policy assignment.";
         public const string NewPolicyAssignmentMetadataHelp = "The metadata for the new policy assignment. This can either be a path to a file name containing the metadata, or the metadata as a string.";
+        public const string NewPolicyAssignmentEnforcementModeHelp = "The enforcement mode for the new policy assignment, e.g. Default, DoNotEnforce. It indicates whether a policy effect will be enforced or not during assignment creation and update. Please visit https://aka.ms/azure-policyAssignment-enforcement-mode for more information.";
```

Comment:
Minor: I think e.g. is not quite the right wording. Maybe "… new policy assignment. Values are Default or DoNotEnforce, indicating whether …"